tuned/profiles/functions/function_intel_recommended_pstate.py

```diff
@@ -0,0 +1,38 @@
+from . import base
+
+processor_name = ("sandybridge",
+        "ivybridge",
+        "haswell",
+        "broadwell",
+        "skylake")
+
+pmu_path = "/sys/devices/cpu/caps/pmu_name"
+active = "active"
+disable = "disable"
```

Comment:
nit: in python, constants are usually in upper-case

Reply:
Done